components/layout/display_list/mod.rs

```diff
@@ -1660,13 +1669,26 @@ fn glyphs(
                     index: glyph.id(),
                     point,
                 };
-                glyphs.push(glyph);
+
+                // First glyph must never be elided. Otherwise, check if it's time to crop.
+                // The first character or atomic inline-level element on a line must be clipped rather than ellipsed.
```

Comment:
This is a quote, so


```suggestion
                // > The first character or atomic inline-level element on a line must be clipped rather than ellipsed.
```